client/rest/licenses/httpasyncclient-LICENSE.txt

```diff
@@ -0,0 +1,558 @@
+                                 Apache License
+                           Version 2.0, January 2004
+                        http://www.apache.org/licenses/
+
+   TERMS AND CONDITIONS FOR USE, REPRODUCTION, AND DISTRIBUTION
```

Comment:
Maybe you should use the license map feature so you only have to list the license for httpclient once? It looks like
```
dependencyLicenses {
  mapping from: /lucene-.*/, to: 'lucene'
}
```